Cylenium/cy.cs

```diff
@@ -245,6 +245,11 @@ public static void Screenshot(string fileName)
 
         #region BROWSER
 
+        public static ReadOnlyCollection<string> WindowHandles()
```

Comment:
A docstring here would be nice. This returns a readonly collection of window handle IDs (strings), but this could be for windows or tabs.